src/services/parser/acknowledgement-parser.js

```diff
@@ -0,0 +1,30 @@
+import { INTERACTION_IDS } from "../../constants/interaction-ids";
+import { XmlParser } from "./xml-parser/xml-parser";
+import { validateFieldsHaveSuccessfullyParsed } from "./parsing-validation";
+
+export const parseAcknowledgementFields = async message => {
```

Comment:
To keep naming consistent, either rename `parseAcknowledgementFields` to `parseAcknowledgementMessage` 
OR
rename `parseContinueRequestMessage` & `parseEhrRequestMessage` to `parseContinueRequestFields` and `parseEhrRequestFields` respectively.